projects/experimental/directives/progress-segmented/progress-segmented.style.less

```diff
@@ -0,0 +1,25 @@
+[tuiProgressBar]._segmented {
+    @radius: calc(var(--t-height) / 2);
```

Comment:
Let's move LESS vars up outside this scope so it's easier to read what is the actual style and what are the helpers.

Reply:
✅